public/test.http

```diff
@@ -0,0 +1,4 @@
+GET https://api.discogs.com/database/search?q=IsThisIt&token=KbEizCiUnPDWhSjCmJvqBZvnzWnlfYtFKEQcGQEf HTTP/1.1
```

Comment:
Definitely don't put this in your public folder. I would just put this in the root of your repository and add it to your gitignore `*.http`